lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -1,56 +1,122 @@
+        #         - If this version is a hotfix (more than 2 parts and 3rd part is non-zero), returns the "X.Y.Z" formatted string
+        #         - Otherwise (not a hotfix / 3rd part of version is 0), returns "X.Y" formatted version number
+        #
         def self.get_public_version
           version = get_release_version
           vp = get_version_parts(version[VERSION_NAME])
-          return "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}" unless is_hotfix(version)
+          return "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}" unless is_hotfix?(version)
           "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}.#{vp[HOTFIX_NUMBER]}"
         end
 
+        # Extract the version name and code from the `vanilla` flavor of the `$PROJECT_NAME/build.gradle file`
+        #   or for the defaultConfig if `HAS_ALPHA_VERSION` is not defined.
+        #
+        # @env HAS_ALPHA_VERSION If set (with any value), indicates that the project uses `vanilla` flavor.
+        #
+        # @return [Hash] A hash with 2 keys "name" and "code" containing the extracted version name and code, respectively
+        #
         def self.get_release_version
           section = ENV["HAS_ALPHA_VERSION"].nil? ? "defaultConfig" : "vanilla {"
           gradle_path = self.gradle_path
-          name = get_version_name_from_file(gradle_path, section)
-          code = get_version_build_from_file(gradle_path, section)
+          name = get_version_name_from_gradle_file(gradle_path, section)
+          code = get_version_build_from_gradle_file(gradle_path, section)
           return { VERSION_NAME => name, VERSION_CODE => code }
         end
 
+        # Extract the version name and code from the `defaultConfig` of the `$PROJECT_NAME/build.gradle` file
+        #
+        # @return [Hash] A hash with 2 keys `"name"`` and `"code"`` containing the extracted version name and code, respectively,
+        #                or `nil` if `$HAS_ALPHA_VERSION` is not defined.
+        #
         def self.get_alpha_version
-          if (ENV["HAS_ALPHA_VERSION"].nil?)
-            return nil
-          end
+          return nil if ENV["HAS_ALPHA_VERSION"].nil?
 
           section = "defaultConfig"
           gradle_path = self.gradle_path
-          name = get_version_name_from_file(gradle_path, section)
-          code = get_version_build_from_file(gradle_path, section)
+          name = get_version_name_from_gradle_file(gradle_path, section)
+          code = get_version_build_from_gradle_file(gradle_path, section)
           return { VERSION_NAME => name, VERSION_CODE => code }
         end
   
-        def self.is_alpha_version(version)
+        # Determines if a version name corresponds to an alpha version (starts with `"alpha-"`` prefix)
+        #
+        # @param [String] version The version name to check
+        #
+        # @return [Bool] true if the version name starts with the `ALPHA_PREFIX`, false otherwise.
+        #
+        #private
```

Comment:
This looks odd in the rendered HTML file – is this supposed to be perhaps:
```suggestion
        # @private
```